src/repositories/job_applicants.js

```diff
@@ -0,0 +1,86 @@
+const knex = requireKnex();
+const baseRepo = requireUtil("baseRepo");
+
+const create = async (payload) => {
+  try {
+    payload = baseRepo.addCreatedTimestamps(payload);
+    let result = await knex.transaction(async (trx) => {
+      const rows = await trx("job_applicants").insert(payload).returning("*");
+      return rows[0];
+    });
+    return result;
+  } catch (error) {
+    throw error;
+  }
+};
+
+const getAllJobApplicants = async (query) => {
+  try {
+    if (query && query.tags && query.tags.length > 0) {
+      var arr = query.tags.split(",");
+      return await getJobApplicantsByTags(arr);
+    } else {
+      const rows = knex("job_applicants").orderBy("created_at", "desc");
+      return rows;
+    }
+  } catch (error) {
+    throw error;
+  }
+};
+
+const findByUuid = async (where = {}) => {
+  return await baseRepo.first("job_applicants", where);
+};
+
+const getAllPostedJobApplicants = async (user_uuid) => {
+  try {
+    const rows = knex("job_applicants").where("user_uuid", user_uuid);
```

Comment:
I think `job_applicants` doesn't have the user_uuid column right? @mahesh-muttinti